Tests/HaCWebsiteLibTests/DateUtilsTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+@testable import HaCWebsiteLib
+
+class DateUtilsTests: HaCWebsiteLibTestCase {
+  func testDate() {
+    let readableDate = Date.from(year: 2017, month: 10, day: 25, hour: 19, minute: 00)
+    let timestampDate = Date(timeIntervalSince1970: 1508954400)
+
+    XCTAssertEqual(readableDate, timestampDate)
+  }
+
+  func testTwiceOccurringDate() {
```

Comment:
I think these tests might be _slightly_ flakey in that if timezones get redefined for some reason, these dates can shift from UTC.

I don't know if that's allowed to happen for past dates :thinking_face: 

Not a blocker though

Reply:
Maybe this is a bridge we should cross when we come to it. Unless there's an obvious alternative way to test this?

Reply:
I can't think of an obvious alternative - I like the idea of waiting for a bridge.